cmd/puppeth/wizard.go

```diff
@@ -42,6 +42,7 @@ import (
 type config struct {
 	path      string   // File containing the configuration values
 	bootnodes []string // Bootnodes to always connect to by all nodes
+	static_bootnode int // Bootnodes count from flags
```

Comment:
use camalCase

Reply:
and lint

Reply:
I think `numBootNodes` would be suitable name for it.

Reply:
Thanks it updated ad https://github.com/Onther-Tech/plasma-evm/pull/98/commits/8cbda99e8ae6939ad6c89928d6f85483c0e275b4